test/test_validation.py

```diff
@@ -463,6 +463,30 @@ def test_invalid_items(self):
             'Incorrectly accepts integers outside of the range'
         )
 
+   def test_valid_character_items(self):
+        self.validate_and_compare(
+            [
+                7,
+                "8",
+                8
+            ],
+            True,
+            "Does not accept integer in a string"
+        )
+
+    # TODO: I was trying to validate [7, "eight", 8], but the test failed.
```

Comment:
Yes, with the way I have tests set up, they either have to all fail, or all pass. If you need a mix of the two then they need to be separate tests (as you've correctly done)

Reply:
perfect